DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs

```diff
@@ -35,5 +40,21 @@ private static string ParseMinutes(int minutes)
             }
             return $"{minutes}m";
         }
+
+        private static string? ParseObjectives(string? objectives)
+        {
+            if (objectives == null)
+            {
+                return null;
+            }
+
+            var doc = new HtmlDocument();
+            doc.LoadHtml(objectives);
+
+            var body = doc.DocumentNode.SelectNodes("//body");
+
+            return body?.FirstOrDefault()?.InnerHtml

```

Comment:
Why the `FirstOrDefault`? Don't we just want everything inside the body? I was imagining something like this https://html-agility-pack.net/from-string but with inner html instead of outer (I assume that will remove the body tags), although I haven't tried that 🙂

Reply:
So this produces a list of bodies, and I'm just taking the first body and returning its contents. Select Single node implements the `FirstOrDefault` for us so I've switched over to using that in 3ff95e1 🙂